components/LocationDetails/index.js

```diff
@@ -6,18 +6,29 @@ import TitleSection from "../TitleSection";
 import NotesForm from "../NotesForm";
 import Note from "../Note";
 import FavoriteButton from "../FavoriteButton";
+import LogInOutButton from "../LogInOutButton";
+import EditMode from "../EditMode";
+import { useState } from "react";
 
 export default function LocationDetails({
   data,
   mutate,
   onToggleLiked,
   isLiked,
+  session
 }) {
+  const [isEditMode, setIsEditMode] = useState(false);
+  const isAdmin = session?.user.name==="HungryOar";
```

Comment:
When you want to use this app after capstone, it would be prudent to run this check in the backend. For example, you could use a `getServersideProps` function where you check for the admin and pass down the result via props to this component. That way you would guard against someone meddling with frontend code (which this component is) and do your critical checks in a save environment where nobody has direct access to.

Reply:
It`s a good idea! Thanks!